docs/user-guide/introduction.rst

```diff
@@ -10,4 +10,13 @@ your own network.
 Another common use case is to copy synced modules into a custom repository and add
 additional modules by uploading them directly. This allows you to test new modules
 or new versions of existing modules and then easily promote them into a production
-repository.
\ No newline at end of file
+repository.
+
+Puppet modules must adhere to the Puppet `3.6+ metadata guidelines
+<https://docs.puppetlabs.com/puppet/latest/reference/modules_publishing.html#publishing-modules-on-the-puppet-forge>`_,
+which require a metadata.json file to be present in the puppet module. Also, each tag.gz file must
```

Comment:
I think metadata.json should be escaped since it's referring to something specific